templates/500.html

```diff
@@ -84,7 +84,6 @@
       <ul>
         <li>DCE/Summer: <a href="mailto:AcademicTechnology@dce.harvard.edu{% if renderable_error %}?subject={{ renderable_error.display_text }}{% endif %}">AcademicTechnology@dce.harvard.edu</a></li>
         <li>FAS: <a href="mailto:atg@fas.harvard.edu{% if renderable_error %}?subject={{ renderable_error.display_text }}{% endif %}">atg@fas.harvard.edu</a></li>
-        {# Note: For icommons_ext_tools, renderable_errors are currently only being shown by django_canvas_course_site_wizard #}
         {# TLT-486 specifies that only DCE and FAS point of contact info will be shown for errors during Canvas course site creation #}
         {% if not renderable_error %}
```

Comment:
I think that unhandled 404s and 500s in qualtrics will now go to this. Rather than rip it all out and leave the default server error messages, which is also a possibility, I'll leave them in here (unless you have concerns).